pquery-run.sh

```diff
@@ -1900,43 +1919,29 @@ pquery_test() {
         if [ $(ls -l ${RUNDIR}/${TRIAL}/*/*core* 2> /dev/null | wc -l) -ge 1 ]; then
-          echoit "mysqld coredump detected at $(ls ${RUNDIR}/${TRIAL}/*/*core* 2> /dev/null)"
-          cd ${RUNDIR}/${TRIAL} || exit 1
-          add_handy_scripts
-          TEXT=$(${SCRIPT_PWD}/new_text_string.sh)
-          echo "${TEXT}" > ${RUNDIR}/${TRIAL}/MYBUG
-          cd - >/dev/null || exit 1
-          if grep -qi "No .* found [ia][nt]" ${RUNDIR}/${TRIAL}/MYBUG; then
-            echoit "Assert: we found a coredump at $(ls ${RUNDIR}/${TRIAL}/*/*core* 2> /dev/null), yet ${SCRIPT_PWD}/new_text_string.sh produced this output: ${TEXT}"
-            exit 1
-          fi
-          echoit "Bug found (as per new_text_string.sh): ${TEXT}"
-          TRIAL_TO_SAVE=1
-          if [ "${ELIMINATE_KNOWN_BUGS}" == "1" -a -r ${SCRIPT_PWD}/known_bugs.strings ]; then # "1": String check hack to ensure backwards compatibility with older pquery-run.conf files
-            set +H                                                                          # Disables history substitution and avoids  -bash: !: event not found  like errors
-            FINDBUG="$(grep -Fi --binary-files=text "${TEXT}" ${SCRIPT_PWD}/known_bugs.strings)"
-            if [ "$(echo "${FINDBUG}" | sed 's|[ \t]*\(.\).*|\1|')" == "#" ]; then FINDBUG=""; fi  # Bugs marked as fixed need to be excluded. This cannot be done by using "^${TEXT}" as the grep is not regex aware, nor can it be, due to the many special (regex-like) characters in the unique bug strings
-            if [ ! -z "${FINDBUG}" ]; then  # do not call savetrial, known/filtered bug seen
-              echoit "This is aleady known and logged, non-fixed bug: ${FINDBUG}"
-              echoit "Deleting trial as ELIMINATE_KNOWN_BUGS=1, bug was already logged and is still open"
-              ALREADY_KNOWN=$[ ${ALREADY_KNOWN} + 1]
-              TRIAL_TO_SAVE=0
-            else
-              NEWBUGS=$[ ${NEWBUGS} + 1 ]
-              echoit "[${NEWBUGS}] *** NEW BUG *** (not found in ${SCRIPT_PWD}/known_bugs.strings, or found but marked as already fixed)"
-            fi
-            FINDBUG=
+          if [[ ${MDG} -eq 1 ]]; then
+            for j in $(seq 1 3); do
+              if [ $(ls -l ${RUNDIR}/${TRIAL}/node${j}/*core* 2> /dev/null | wc -l) -ge 1 ]; then
+                export GALERA_ERROR_LOG=${RUNDIR}/${TRIAL}/node${j}/node${j}.err
+                export GALERA_CORE_LOC=$(ls -t ${RUNDIR}/${TRIAL}/node${j}/*core* 2> /dev/null)
+                echoit "mysqld coredump detected at $(ls ${RUNDIR}/${TRIAL}/node${j}/*core* 2> /dev/null)"
+                handle_bugs
```

Comment:
Does handle_bugs handle dups detection (according to new_text_string) for all three nodes?

Reply:
yes, `handle_bugs` function will verify each node separately to check the duplicate bugs. But we need to update `add_handy_scripts` function to add gdb script for each node.

Reply:
ack